src/pudl/package_data/eia923/column_maps/generation_fuel.csv

```diff
@@ -95,4 +95,3 @@ total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtus,total_fuel_consumptio
 elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu
 total_net_generation_mwh,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours
 report_year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year
-early_release,,,,,,,,,,,,,,,,,,,,,,,early_release_data_july_2024_not_fully_edited_use_with_caution_do_not_aggregate_to_state_regional_or_national_totals,
```

Comment:
Do we want to delete this or just keep it blank?

Reply:
Leaving it blank means we wind up carrying around a completely empty column in the raw tables. Is there any reason to do so?

Reply:
I don't have super strong feelings about this? Might make adding the early release data a bit easier to add because we won't forget about these columns.

Reply:
I was also thinking for consistency with the byproduct_expenses table that does have a weird value for early_release in 2009 and thus an early_release column that sticks around all the time.

Reply:
That's fair, also consistent with how it was prior to the ER integration PR (#3721). I'll revert it.